site/en/guide/keras/train_and_evaluate.ipynb

```diff
@@ -1575,7 +1575,7 @@
         "# Instantiate an optimizer.\n",
         "optimizer = keras.optimizers.SGD(learning_rate=1e-3)\n",
         "# Instantiate a loss function.\n",
-        "loss_fn = keras.losses.SparseCategoricalCrossentropy()\n",
+        "loss_fn = keras.losses.SparseCategoricalCrossentropy(from_logits=True)\n",
```

Comment:
The last layer activation of the model is set to 'softmax'. Can you remove that so that from_logits=True will work?